### ISSUE TITLE
create_batch() / build_batch() TypeError for models with a 'size' attribute

### ISSUE THREAD

**Issue by adamchainz** [FactoryBoy/factory_boy#901]
#### Proposed solution
*Please provide some wild idea you think could solve this issue. It's much easier to work from an existing suggestion :)*

#### Extra notes
*Any notes you feel interesting to include: alternatives you've considered, reasons to include the change, anything!*

**Comment by rbarrois** [comment]
Indeed, a workaround would be possible, the simplest one would be [`Meta.rename`](https://factoryboy.readthedocs.io/en/stable/reference.html#factory.FactoryOptions.rename):
```python
class ContainerFactory:
  class Meta:
    model = Container
    rename = {
      "size_": "size"
    }
```

And then calling `ContainerFactory.create_batch(3, size_=5)`

Nevertheless, I agree with the initial issue; it would however be somewhat complicated to change, as that pattern (`create_batch(size=...`) is already used by lots of user code.
